src/screens/PickList/index.tsx

```diff
@@ -317,6 +340,11 @@ const PickOrderItem = (props: any) => {
                         />
                       )}
                       <Button title="Pick Item" onPress={() => formSubmit(item.id)} disabled={!item?.quantityRemaining} />
+                      <Button
+                          title="Revert"
+                          onPress={() => revertPickItem(item.id)}
+                          disabled={item?.quantityRemaining !== 0}
```

Comment:
I think it should be enabled even if partially picked, when `quantity remaining < quantity to pick`. Additionally, I think you could hide the button if `quantity remaining === quantity to pick`.